File: java/socketd/src/main/java/org/noear/socketd/exception/SocketdCodecException.java
package org.noear.socketd.exception;

/**
 * 编码异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketdCodecException extends RuntimeException {
    public SocketdCodecException(Throwable cause){
        super(cause);
    }

    public SocketdCodecException(String message){
        super(message);
    }
}

File: dotnet/socketd/org.noear.socketd/exception/SocketdCodecException.cs
namespace org.noear.socketd.exception;

/**
 * 编码异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketdCodecException : SocketdException
{
 public SocketdCodecException(string message) : base(message)
 {

 }

 public SocketdCodecException(String message, Exception cause) : base(message, cause)
 {

 }
}